src/AddTodoForm.jsx

```diff
@@ -1,8 +1,15 @@
-function AddTodoForm () {
+function AddTodoForm (props) {
+    function handleAddTodo(event) {
+       event.preventDefault();
+       let todoTitle = event.target[0].value;
+       props.onAddTodo(todoTitle);
+       console.log(todoTitle);
+       event.target[0].value ='';
+    }
  return(
-    <form>
+    <form onSubmit={(handleAddTodo)}>
         <label htmlFor="todoTitle">Title</label>
-        <input id="todoTitle"></input>
+        <input attribute="title" id="todoTitle"></input>
```

Comment:
This should be `name="title"` (the directions here are confusingly wordy). `attribute` is not a valid property that you can put on an input. My IDE flags this with a red squiggle that says `Unknown property 'attribute'`.